tools/docker/tests/test_gw_repeater.sh

```diff
@@ -60,6 +60,9 @@ main() {
         sleep ${DELAY}
         status "Start Repeater (Remote Agent)"
         ${scriptdir}/../run.sh ${VERBOSE_OPT} ${FORCE_OPT} start-agent -d -n ${REPEATER_NAME} -m aa:bb:cc:dd "$@"
+        
+        status "Start Repeater2 (Remote Agent2)"
+        ${scriptdir}/../run.sh ${VERBOSE_OPT} ${FORCE_OPT} start-agent -d -n ${REPEATER2_NAME} -m aa:bb:cc:ee "$@"
```

Comment:
I would prefer that there would be a command-line option that specifies the number of agents, and that they're started in a loop. The MAC address can be set to `aa:bb:cc:$n$n` where n is the iterator.

Reply:
Agree with Arnout, we can update the `-r|--repeater` argument as follows:
```
while true; do
    case "$1" in
        ...
        -r | --repeater) REPEATER_NAME="$REPEATER_NAME $2"; shift; shift ;;
        ...
    esac
done
```

And then create multiple repeaters by iterating through `REPEATER_NAME` as follows:
```
for repeater in $REPEATER_NAME
do
    # Start repeater with $repeater name
done
```